src/pint/residuals.py

```diff
@@ -561,8 +580,18 @@ class CombinedResiduals(object):
     def __init__(self, residuals):
         self.residual_objs = collections.OrderedDict()
         for res in residuals:
+            res._is_combined = True
             self.residual_objs[res.residual_type] = res
 
+    @property
+    def model(self):
```

Comment:
This doesn't work for users who expect `residual.model` to be a TimingModel they can query for, say, `free_params`. If you want to provide this differently typed object, please call it something else; if you don't want to use a single TimingModel object for combined residuals, just leave `.model` raising AttributeError.